app.js

```diff
@@ -0,0 +1,36 @@
+let arrayOfTodos = []
+
+const fetchTodos = () => {
+  fetch('https://jsonplaceholder.typicode.com/todos')
+    .then((response) => response.json())
+    .then((json) => arrayOfTodos = json)
+}
+
+const logTodos = () => {
+  console.log(arrayOfTodos)
+}
+
+const populateTodos = () => {
+  for (let i = 0; i < arrayOfTodos.length; i++) {
+    let e = arrayOfTodos[i];
+    let li = document.createElement('li');
+
+    let userId = "UserID = " + e.userId + ', ';
+    let id = "ID = " + e.id + ', ';
+    let title = 'Title = ' + e.title + ', ';
+    let completed = "Completed = " + e.id;
```

Comment:
another way to do this is using javascript es6 method called Template Literals.
you use backticks `` and placeholders for values: `${insert-value}`
ex: let userId = `UserID = ${e.userId},`